dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3PutJob.java

```diff
@@ -84,10 +67,25 @@ public Ds3PutJob(final Ds3Client client, final List<File> files, final String bu
         this.settings = settings;
         final Stage stage = (Stage) ALERT.getDialogPane().getScene().getWindow();
         stage.getIcons().add(new Image(ImageURLs.DEEPSTORAGEBROWSER));
+
+    }
+
+    public SettingsStore getSettings() {
+        return settings;
+    }
+
+    public Ds3Client getClient() {
+        return client;
+    }
+
+    public DeepStorageBrowserPresenter getDeepStorageBrowserPresenter() {
+        return deepStorageBrowserPresenter;
     }
 
     @Override
     public void executeJob() throws Exception {
+        //job start time
+        final Calendar jobStartTime = Calendar.getInstance();
```

Comment:
The new java 8 time libraries should be used rather than the old pre java 8 time libraries.